specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/JobPrivateEndpoints.json

```diff
@@ -0,0 +1,329 @@
+{
+  "swagger": "2.0",
```

Comment:
Please fix the Avocado error: [https://github.com/Azure/azure-rest-api-specs/pull/24902/checks?check_run_id=15311430531](https://github.com/Azure/azure-rest-api-specs/pull/24902/checks?check_run_id=15311430531).

Reply:
Avocado error is only because there isn't an existing 2023-05-01-preveiw dev branch yet. This PR is just to run the validation report. Once the 2023-05-01-preveiw dev branch is created, I'll point these changes to that branch and the avocado error should go away.